Userland/Libraries/LibJS/Bytecode/Pass/LoadElimination.cpp

```diff
@@ -181,16 +180,20 @@ static NonnullOwnPtr<BasicBlock> eliminate_loads(BasicBlock const& block, size_t
 void EliminateLoads::perform(PassPipelineExecutable& executable)
 {
     started();
+    static HashMap<u32, Register> register_rerouting_table {};
```

Comment:
We shouldn't need to call `clear()` right? Since it's being defined to an empty map on every call to preform.

Reply:
I'd moved it up out of both functions but this should be fine as well,
note that #20122 will make this effort for naught